DogeNet.BLL/Features/Conversations/AddParticipantConversation/AddParticipantConversationHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿// <copyright file="AddParticipantConversationHandler.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.BLL.Features.Conversations.AddParticipantConversation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using DogeNet.DAL;
+    using DogeNet.DAL.Models;
+    using MediatR;
+
+    public class AddParticipantConversationHandler : IRequestHandler<AddParticipantConversationCommand>
+    {
+        private readonly DBContext context;
+
+        private readonly IMapper mapper;
+
+        public AddParticipantConversationHandler(DBContext context, IMapper mapper)
+        {
+            this.context = context;
+            this.mapper = mapper;
+        }
+
+        public Task<Unit> Handle(AddParticipantConversationCommand request, CancellationToken cancellationToken)
+        {
+            var participantConversation = this.mapper.Map<ConversationParticipant>(request.model);
+
+            participantConversation.AddedAt = DateTime.Now;
+
+            this.context.ConversationParticipants.Add(participantConversation);
+            this.context.SaveChanges();
```

Comment:
await for changes?

Reply:
![image](https://user-images.githubusercontent.com/77879337/145009737-af49625c-8262-48ef-829c-f0bd0be64bbe.png)
done